packages/react-components/react-menu/src/components/Menu/Menu.types.ts

```diff
@@ -162,6 +162,9 @@ export type MenuOpenChangeData = {
   bubble?: boolean;
   /**
    * Indicates whether the change of state was a keyboard interaction
+   * @deprecated
+   * This component will return false always,
```

Comment:
is it possible to return the correct value still ? returning false always is still a breaking change if anyone relies on this property.

Also `Escape` shouldn't be interpreted as a click right ? but that information you can get from the actual event

Reply:
Sounds right, this would be a breaking change. I'll revert this